docs/reference/expertfeatures.rst

```diff
@@ -0,0 +1,123 @@
+Inmanta Expert Features
+=======================
+
+This page documents the set of expert features for the Inmanta application and extensions.
+
+.. attention::
+
+        "With great power comes great responsibility"
+
+    These features have to be used with extreme caution. Make sure you are fully aware of what they do before using
+    them. These feature allow you to bypass safety checks that the product offers and/or that were put in place by
+    the model developers.
+
+Compiler expert features
+########################
+
+Allow handler code update during a partial compile
+--------------------------------------------------
+
+
+    By default, updating handler code is only allowed during full compiles. A
+    safety check is performed during partial compiles to make sure that inmanta
+    module code remains consistent across partial versions.
+
+    If a full compile cannot be performed and handler code update is necessary,
+    it is possible to disable the version consistency check and to allow handler
+    code update in a manual export as follows:
+
+
+
+    .. danger::
+        Make sure the updated handler code is compatible with all resource
+        schemas it might have to process. Uploading incompatible handler code can lead
+        to very hard to diagnose issues.
```

Comment:
```suggestion
        The updated code will be used for all handlers and resource, even if they are not in this partial compile.
        Make sure the updated handler code is compatible with all resource
        schemas it might have to process. Uploading incompatible handler code can lead
        to very hard to diagnose issues.
```